srcpkgs/python3-dkimpy/template

```diff
@@ -22,6 +22,6 @@ post_extract() {
 
 post_install() {
 	# This file is used for testing only
-	rm -f ${DESTDIR}/usr/lib/python3.?/site-packages/dkim/__main__.py
+	rm ${DESTDIR}/usr/lib/python3.?/site-packages/dkim/__main__.py
```

Comment:
Why make this change? There is no need to break the build if this file disappears. Also, while you're editing this line, might as well make it
```bash
rm -f ${DESTDIR}/${py3_sitelib}/dkim/__main__.py
```

Reply:
I prefer to break the build, because if upstream stops installing that file we can just clean up the line, and we will realize that the line can be cleaned up at all. Otherwise it can stay there silently even when it no longer applies.

Reply:
Ok, but I still prefer the `$py3_sitelib` change. Less typing and no globs.

Reply:
Sure, I definitely agree with that part.

Reply:
thanks